modules/nomad-module/variables.tf

```diff
@@ -0,0 +1,55 @@
+variable "ssh_key" {
```

Comment:
Can we define some reasonable defaults to the values in this file so users don't have to specify everything?